vllm/entrypoints/openai/api_server.py

```diff
@@ -553,13 +554,20 @@ async def fake_stream_generator() -> AsyncGenerator[str, None]:
                         type=json.loads,
                         default=["*"],
                         help="allowed headers")
+    parser.add_argument("--served-model-name",
+                        type=str,
+                        default=None,
+                        help="The model name used in the API. If not "
+                        "specified, the model name will be the same as "
+                        "the huggingface name.")
```

Comment:
A dumb question: What is this for? How is it different from `args.model`?

Reply:
This is an alias used for client querying API. For example, the server manager can specify `"lmsys/vicuna-13b-v1.3"` as just `vicuna-13b` for simplicity